### PR TITLE
Copy template modal missing tip

### PR DESCRIPTION
**Pull request by infograf768**
Edit a template (not a template style). Click on `Copy Template` toolbar button. No tip.
Patch.
Result should be:
screen shot 2016-04-24 at 10 25 50

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 735666fb75d1e48ff6cc71bcc451121681bca595<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10071">issues.joomla.org/joomla-cms/10071</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 735666fb75d1e48ff6cc71bcc451121681bca595<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10071">issues.joomla.org/joomla-cms/10071</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10071">issues.joomla.org/joomla-cms/10071</a>.</sub>

File: administrator/components/com_templates/views/template/tmpl/default_modal_collapse_body.php [hunk 1]
```diff
@@ -8,10 +8,14 @@
  */
 
 defined('_JEXEC') or die;
+
+JHtml::_('bootstrap.tooltip', '.modalTooltip', array('container' => '#collapseModal'));
 ?>
 <div id="template-manager-css" class="form-horizontal">
-	<div class="control-group">
-		<label for="new_name" class="control-label hasTooltip" title="<?php echo JHtml::tooltipText('COM_TEMPLATES_TEMPLATE_NEW_NAME_DESC'); ?>"><?php echo JText::_('COM_TEMPLATES_TEMPLATE_NEW_NAME_LABEL')?></label>
+	<div class="control-group span6">
```
Comment (JoomliC) [inline administrator/components/com_templates/views/template/tmpl/default_modal_collapse_body.php]: Why the span 6 ?

File: administrator/components/com_templates/views/template/tmpl/default_modal_collapse_body.php [hunk 2]
```diff
@@ -8,10 +8,14 @@
...
+		<label for="new_name" class="modalTooltip" title="<?php echo JHtml::_('tooltipText', 'COM_TEMPLATES_TEMPLATE_NEW_NAME_LABEL', 'COM_TEMPLATES_TEMPLATE_NEW_NAME_DESC'); ?>">
```
Comment (JoomliC) [inline administrator/components/com_templates/views/template/tmpl/default_modal_collapse_body.php]: In fact, you can just embueded the label in the div control-label:
```
		<div class="control-label">
			<label for="new_name" class="modalTooltip" title="<?php echo JHtml::_('tooltipText', 'COM_TEMPLATES_TEMPLATE_NEW_NAME_LABEL', 'COM_TEMPLATES_TEMPLATE_NEW_NAME_DESC'); ?>">
				<?php echo JText::_('COM_TEMPLATES_TEMPLATE_NEW_NAME_LABEL')?>
			</label>
		</div>
```

This way, the label is vertically align with the input (issue as shown on your screenshot) ;-)

**Comment by JoomliC** [comment]
@infograf768 The span6 gives trouble for the label, not vertically aligned with the input.
I have commented your code, as with a simple div control-label to include the label, it solves this display issue ;-)

**Comment by infograf768** [comment]
I added the span6 because of the position of the tip.
If I follow you (as far as I understand) and change to:
```
<div id="template-manager-css" class="form-horizontal">
	<div class="control-label">
		<label for="new_name" class="modalTooltip" title="<?php echo JHtml::_('tooltipText', 'COM_TEMPLATES_TEMPLATE_NEW_NAME_LABEL', 'COM_TEMPLATES_TEMPLATE_NEW_NAME_DESC'); ?>">
			<?php echo JText::_('COM_TEMPLATES_TEMPLATE_NEW_NAME_LABEL')?>
		</label>
	</div>
	<div class="controls">
		<input class="input-xlarge" type="text" id="new_name" name="new_name"  />
	</div>
</div>
```

then I get:

screen shot 2016-04-25 at 08 34 41

**Comment by infograf768** [comment]
BTW, the tip here is now displayed in the same position as in Batch modals, where it does so grace of the span6.

**Comment by JoomliC** [comment]
@infograf768 ok i understand your intention ;-)
But my proposal was not exactly has you supposed, but using bootstrap standard form field structure like this : 
```
<div id="template-manager-css" class="form-horizontal">
    <div class="control-group">
        <div class="control-label">
            <label for="new_name" class="modalTooltip" title="<?php echo JHtml::_('tooltipText', 'COM_TEMPLATES_TEMPLATE_NEW_NAME_LABEL', 'COM_TEMPLATES_TEMPLATE_NEW_NAME_DESC'); ?>">
                <?php echo JText::_('COM_TEMPLATES_TEMPLATE_NEW_NAME_LABEL')?>
            </label>
        </div>
        <div class="controls">
            <input class="input-xlarge" type="text" id="new_name" name="new_name"  />
        </div>
    </div>
 </div>
```
capture d ecran 2016-04-25 a 11 46 11



If you want same display as inside batch (which is loaded field from layout and not constructing it as in copy modal) so you should not use form-horizontal.


There the code with exactly the batch modal rendering : 
```
<div id="template-manager-css" class="row-fluid">
	<div class="control-group span6">
		<div class="controls">
			<label for="new_name" class="control-label modalTooltip" title="<?php echo JHtml::tooltipText('COM_TEMPLATES_TEMPLATE_NEW_NAME_DESC'); ?>"><?php echo JText::_('COM_TEMPLATES_TEMPLATE_NEW_NAME_LABEL')?></label>
			<input class="input-xlarge" type="text" id="new_name" name="new_name"  />
		</div>
	</div>
</div>
```
capture d ecran 2016-04-25 a 11 43 44

@infograf768 as your PR is already merged, no need to revert it, as i've planned to change name of modal id (which should be unique) from collapseModal to copyModal and so be logical with other modals of this view (renameModal, fileModal, folderModal...) so i could do the change in the same time! ;-)

So, what's best ?
- Batch look like (with span6)
- Or Bootstrap form field look like (as for all edit forms)

**Comment by infograf768** [comment]
@JoomliC 
Cyril, I honestly don't care, as long as tips are readable and interface is consistent. :smiley:

**Comment by JoomliC** [comment]
@infograf768 :+1: 
I will check it to improve it, and in the same time, display hidden tooltip in some othe modals of this view (the more i work on modals, the more i see minor issues, so good to fix them all)
Will inform there when PR will be ready ;-)

**Comment by JoomliC** [comment]
@infograf768 just created the PR https://github.com/joomla/joomla-cms/issues/10100 with a few improvement not only for tooltip, but too for usuability of file and folder modals ;-)

**Pull request merged**